tendrl/commons/jobs/__init__.py

```diff
@@ -104,6 +105,10 @@ def process_job(job):
                     _msg = str("Timed-out (>10min as 'new')")
                     job.errors = _msg
                     job.save()
+                    alert_utils.alert_job_status(
+                        "failed",
+                        "Job %s timed out" % jid
```

Comment:
msgs which contain any ID's or uuids should be framed like below

"Job timed out (job_id: %jid)" 

same for other msgs below

Reply:
Done

Reply:
"Job timed out (job_id: %jid)"

This is how the msg should look like, I dont see it changed, please change it